app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @current_user = current_user
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @current_user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.create(post: @post, author: @current_user, text: params[:comment][:text])
```

Comment:
- [ Optional ] I would like to suggest that you use strong parameters in your controllers when expecting input from `forms` this will prevent assigning request parameters to objects unless they have been explicitly permitted by your controller. Please see this article for more details on this [Click Here >>](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

## Example code
```ruby
   class PostsController < ApplicationController
     
      def create
        @post = @current_user.posts.new(post_params)

        respond_to do |format|
          if @post.save
            format.html { redirect_to @post }
          else
            format.html { render :new }
          end
        end
      end

    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end

  end
```

Reply:
Thank you for the advice, I will read the article.